_layouts/event.html

```diff
@@ -12,7 +12,9 @@ <h1 class="post-title">{{ page.title | escape }}</h1>
 
     <h2>{% translate global.code_of_conduct.title %}</h2>
     {% translate global.code_of_conduct.summary %}
-    <h2>{% translate connect.info %}</h2>
-    {% include links.html %}
+    {% unless page.hide_form %}
+      <h2>{% translate connect.info %}</h2>
+      {% include links.html %}
```

Comment:
ideally links.html would separate out the social media icons which are purely display, which we could keep, and the offending signup form which itself is what we wanna hide here.